Singletons/HighlightUI.gd

```diff
@@ -6,10 +6,12 @@ extends Node
 # start_highlight() and stop_highlight() to highlight
 # registered targets.
 
+# When user does a specific action to acknowledge the highlighted
+# area, this signal should be emitted.
+signal highlight_target_ack(highlight_target: String)
```

Comment:
Move this signal to EventBus and rename to something like "tutorial_action_complete". This signal is not specific to highlight, because it's also called when towers are built and items are moved.

Reply:
Done